bazel/external/lightstep.BUILD

```diff
@@ -1,25 +1,21 @@
 load("@envoy//bazel:genrule_repository.bzl", "genrule_cc_deps", "genrule_environment")
 load(":genrule_cmd.bzl", "genrule_cmd")
 
+_HDRS = glob([
+    "src/c++11/lightstep/**/*.h",
+    "src/c++11/mapbox_variant/**/*.hpp",
+])
+
+_PREFIX_HDRS = [hdr.replace("src/c++11/", "_prefix/include/") for hdr in _HDRS] + [
```

Comment:
I guess this kind of hackery is worth it for the globs, but this goes back to my original rationale for preferring a `repository_rule` for the full build over `genrule`; you can know all the final headers and glob them in place.

This comment is really non-actionable, but I think as we move more stuff to be `genrule` based, we should look to see what happens with this.